e2e/jest.test-setup.ts

```diff
@@ -6,8 +6,9 @@ const userAgent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/5
 beforeEach(async () => {
   await page.setUserAgent(userAgent);
   // Refer to https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetdefaultnavigationtimeouttimeout
-  await page.setDefaultNavigationTimeout(90000);
-  await page.setDefaultTimeout(60000);
+  await page.setDefaultNavigationTimeout(70000);
```

Comment:
There are lots of places throughout where you have timeouts configured. Could they be consolidated here?

Additionally, if we could keep track of places where we "wasted" extra timeout time, then we might be able to find the long tent poles to performance. But if the worst offenders encourage us to increase all of them, then we might not have very tight bounds on things.

In other words, if we tracked how long every step of the e2e suite takes, we could potentially pinpoint regressions.